src/use-document-exec-with-formatBlock.hook.js

```diff
@@ -0,0 +1,20 @@
+import {useContext} from 'react'
+import {RichTextContext} from './rich-text-container.component.js'
+
+export function useDocumentExecWithFormatBlock() {
+  const richTextContext = useContext(RichTextContext)
+  const commandName = 'formatBlock'
+
+  return {
+    performCommandWithFormatBlock(value) {
+      richTextContext.selectRangeFromBeforeBlur()
+      const showDefaultUI = null
+      if (value === document.queryCommandValue(commandName)) {
+        document.execCommand(commandName, showDefaultUI, 'div')
```

Comment:
Instead of doing this, let's reuse `useDocumentExecCommand`. That way we don't have to reimplement the richTextContext.selectRangeFromBeforeBlur and anything else that's in there